drivers/platform/stm32/stm32_tdm.c

```diff
@@ -129,6 +132,10 @@ int32_t stm32_tdm_init(struct no_os_tdm_desc **desc,
 		goto error;
 		break;
 	};
+	if (param->rx_complete_callback) {
```

Comment:
you don't need to check, you can simply copy it, if it's NUL you will anyway detect it later when checking `desc->rx_complete_callback`

Reply:
Removed the check here